### PR TITLE
PST-470: update to gem spec and version

### PR DESCRIPTION
**Pull request by Sitzilla**
@kennycason should be fairly straightforward. I created a new gem here:
https://rubygems.org/gems/datarank-v2

### CODE REVIEW

**Comment by kennycason** [comment]
@Sitzilla 👍  Also, minor note, but no need to place our internal JIRA Ids on a public repo.

File: datarank.gemspec [hunk 1]
```diff
@@ -4,10 +4,10 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'datarank/version'
 
 Gem::Specification.new do |gem|
-  gem.name          = "datarank"
+  gem.name          = "datarank-v2"
   gem.version       = Datarank::Version.to_s
-  gem.authors       = ["Addam Hardy","Evan Sitzes"]
```
Comment (dempe) [inline datarank.gemspec]: lol

File: lib/datarank/version.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module Datarank
   # A class for managing Datarank's Version
   class Version
     MAJOR = 1
-    MINOR = 5
```
Comment (dempe) [inline lib/datarank/version.rb]: why are you going from 5 to 0? o_O

Comment (Sitzilla) [reply lib/datarank/version.rb]: after discussing with @kennycason we decided to take it back to 1.0.0.  Ad you can see we are creating a new gem, so it makes sense to start fresh

**Pull request closed without merging**